### PR TITLE
feat & refactor

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Added Static Input Validation Components.
- Added Side-Nav links and adjusted Nav bar.

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    display: block;
+    border: 3px solid transparent;
+    padding: 0.7rem;
+    width: 100%;
+}
+
+.form-control label {
+    font-size: small;
+    display: inline-block;
+    margin: 15px 0px 5px 0px;
```
Comment (shubhamkumar648) [inline Components/Input/input.css]: you should always prefer rem instead of px for better web performence

**Review by shubhamkumar648 — APPROVED** [review]
good use of the class name, see my comment as a suggestion! remaining code looks clean

File: Components/Input/input.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Components/Input/input.css]: Oh okay Thankyou, will make the change

File: Components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: var(--prime-blue);
+    font-family: 'Open Sans', sans-serif;
+    padding: 2rem;
+    border-radius: 4px;
+    width: 300px;
+}
+
+.header {
+    padding: 1rem 2rem 1rem 0rem;
+    font-size: x-large;
```
Comment (tewarimohit) [inline Components/Input/input.css:12]: Try to avoid hard coding if possible..

**Review by tewarimohit — APPROVED** [review]
Code looks Good to me... just go through suggestions..

File: Components/Input/input.css (hunk 2 above)
Comment (MaruthiPothuganti) [reply Components/Input/input.css:12]: Will try to avoid. Thankyou for the suggestion.

**Pull request merged**